utils/airdropWindows.ts

```diff
@@ -54,23 +49,23 @@ export const findActiveWindow = (
         windowA.airdrop_window_id - windowB.airdrop_window_id
     );
 
-  const activeWindow = sortedWindows.find((windowA) => {
-    return (
-      checkDateIsGreaterThan(
-        windowA.airdrop_window_registration_start_period
-      ) ||
-      checkDateIsGreaterThan(windowA.airdrop_window_registration_end_period) ||
-      checkDateIsGreaterThan(windowA.airdrop_window_claim_start_period) ||
-      checkDateIsGreaterThan(windowA.airdrop_window_claim_end_period)
-    );
-  });
+  const activeWindow = sortedWindows.find((windowA) => (
+      checkDateIsGreaterThan(windowA.airdrop_window_registration_start_period, moment()) ||
```

Comment:
@raamb create one `moment()` instance and assigns it to a variable to avoid unnecessary initialization.

Reply:
Will fix this in my next PR